test/crv-staking/incentivized-voting-lockup-gov.spec.ts

```diff
@@ -0,0 +1,874 @@
+/* eslint-disable @typescript-eslint/dot-notation */
+/* eslint-disable no-await-in-loop */
+/* eslint-disable @typescript-eslint/naming-convention */
+const moduleAlias = require('module-alias')
```

Comment:
what other tests you think are relevant
https://github.com/mstable/mStable-contracts/tree/master/test